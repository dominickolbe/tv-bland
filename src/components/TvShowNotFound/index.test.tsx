import { shallow } from "enzyme";
import React from "react";
import { TvShowNotFound } from "./";

describe("<TvShowNotFound />", () => {
  it("renders snapshots", () => {
    const wrapper = shallow(<TvShowNotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
