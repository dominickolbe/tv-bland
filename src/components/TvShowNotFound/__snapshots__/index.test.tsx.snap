// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TvShowNotFound /> renders snapshots 1`] = `
<WithStyles(ForwardRef(Grid))
  item={true}
  xs={12}
>
  <WithStyles(ForwardRef(Typography))
    color="textSecondary"
    component="p"
    variant="body2"
  >
    TvShowNotFound
  </WithStyles(ForwardRef(Typography))>
</WithStyles(ForwardRef(Grid))>
`;
